src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -3,12 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from __future__ import absolute_import
```

Comment:
Why do you need this?

Reply:
Thanks for pointing this out. This was left by mistake. Removed.